frontend/integration-tests/views/storage-dashboard.view.ts

```diff
@@ -0,0 +1,9 @@
+import { $, $$ } from 'protractor';
+
+export const clusterHealth = $('[class="co-dashboard-text--small co-health-card__text"]');
+export const detailsCardStructure = $$('.co-details-card__body dt');
+const clusterDetails = $$('.co-details-card__body dd');
+export const serviceName = clusterDetails.get(0);
```

Comment:
We should create a ticket to add `data-test-id` attributes to the cards, to avoid using indexes, because when structure changes, this will stop working. Not a dealbreaker atm in my opinion, but should be tracked.